openedx/core/djangoapps/course_groups/migrations/rerun_0006.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [ $# -eq 0 ]; then
+    echo "$0: usage: rerun_0006.sh <arguments>. At minimum, '--settings=<environment>' is expected."
+    exit 1
+fi
+
+./manage.py lms migrate course_groups 0005 --fake "$@"
+./manage.py lms migrate course_groups 0006 "$@"
```

Comment:
Why is that? (I'm not super well-versed in bash scripting in general).

Each positional argument should be split into its own separate word: http://www.gnu.org/software/bash/manual/bash.html#Special-Parameters